src/Generator/AttributeGenerator/AttributeAssembler.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Laminas\Code\Generator\AttributeGenerator;
+
+use Stringable;
+
+interface AttributeAssembler extends Stringable
```

Comment:
Let's not make this `Stringable`: `Stringable` is a terrible API. 

See also https://github.com/ShittySoft/symfony-live-berlin-2018-doctrine-tutorial/pull/3#issuecomment-460441229 for more clarity